app/main.py

```diff
@@ -1,3 +1,62 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.car import Car
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+
+    with open("app/config.json", "r") as file:
+
+        data = json.load(file)
+        fuel_price = data["FUEL_PRICE"]
+        customers = data["customers"]
+
+        customers_instances = []
+        for customer in customers:
+            cus = Customer(customer["name"],
+                           customer["product_cart"],
+                           customer["location"],
+                           customer["money"],
+                           Car(customer["car"]["brand"],
+                               customer["car"]["fuel_consumption"]
+                               ))
```

Comment:
Rewrite using unpacking `**customer`
You can place Car creation inside `Customer.__init__`